tornado/test/netutil_test.py

```diff
@@ -96,6 +96,28 @@ def tearDown(self):
         super(BlockingResolverErrorTest, self).tearDown()
 
 
+@skipIfNoNetwork
+@unittest.skipIf(futures is None, "futures module not present")
+class OverrideResolverTest(AsyncTestCase, _ResolverTestMixin):
+    def setUp(self):
+        super(OverrideResolverTest, self).setUp()
+        mapping = {
+            ('localhost', 80): ('127.0.0.1', 80),
+            ('localhost', 80, socket.AF_INET): ('127.0.0.1', 80),
+            ('localhost', 80, socket.AF_INET6): ('::1', 80)
```

Comment:
These are exactly the results you'd expect, so how can you tell that the OverrideResolver is doing anything? The override should do something like mapping `google.com` to `1.2.3.4`.

Reply:
Ok. Sounds reasonable. I didn't think to test ```OverrideResolver``` itself, instead I only tried to test multiple ips here. But your idea seems to be more general. I'll fix it, thanks.